src/pages/Front/front.scss

```diff
@@ -1,4 +1,39 @@
-@import '../../assets/sass/main.scss';
+@import "../../assets/sass/main.scss";
+
+.front-page {
+  background-color: white;
+
+  &__logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 68rem;
+    position: relative;
+    overflow: hidden;
+  }
+  &-button {
```

Comment:
You can create a Button component and use it in both pages( front and login)

Reply:
Thanks for that. I have created it and made new commits